index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+   <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- Navbar -->
+
+    <header>
+       <div class="nav-container">
+        <div class="bars">
+            <div class="time">
+                <img src="./images/↳-Time.png" alt="Picture of the current time">
+            </div>
+               <div class="bar">
+                   <img src="./images/Group-3.png" alt="">
+               </div>
+        </div>
```

Comment:
**[REQUIRED CHANGES]**

- Great job adding these features but I think the template from Figma was viewed in phone that's why that section was shown. Please, kindly remove it as it's not necessary.

Reply:
Okay, I will work on it.